spatialprofilingtoolbox/db/schema_infuser.py

```diff
@@ -66,3 +69,58 @@ def verbose_sql_execute(self, filename_description,
                             **kwargs):
         verbose_sql_execute(filename_description, self.get_connection(),
                             source_package=source_package, **kwargs)
+
+    def link_fraction_views(self):
+        """
+        Transcribe phenotype fraction features in features system
+        """
+        connection = self.get_connection()
+        cursor = connection.cursor()
+        feat_extr_query = """
+            SELECT t1.marker_symbol, t1.measurement_study, t1.average_percent, t1.stratum_identifier, t2.sample
+            FROM fraction_stats  t1
+            LEFT JOIN (
```

Comment:
What's the reason to LEFT JOIN on the `sample_strata` table?
Also why are we selecting from `fraction_stats` and not `fraction_generalized_cases` (see comment in #121)?
It seems you are artificialy adding a `sample` column by joining on `sample_strata`, but the result will be that the `average_percent` values are equal for all samples. `fraction_stats` is statistics, i.e. averages, not the feature values themselves.

Reply:
It should actually be one "level" up, i.e. the view: `fraction_by_marker_study_specimen`.
`fraction_generalized_cases` is artifically duplicated with "null" stratum identifier, so that there is 1 aggregation case over the whole cohort (and so not relevant to us here).